app/domain/authentication/audit_log.rb

```diff
@@ -1,17 +1,35 @@
 # frozen_string_literal: true
 
 module Authentication
-  class AuditLog
-    def self.record_authn_event(role:, webservice_id:, authenticator_name:,
-                                success:, message: nil)
+
+  AuditLog = CommandClass.new(
```

Comment:
`CommandClass` names should be verb phrases.  Not sure what's most appropriate here, maybe something like `WriteAuditEvent`?